src/Launcher.cpp

```diff
@@ -1,10 +1,10 @@
-#include "launcher.hpp"
+#include "Launcher.hpp"
 
-#include <map>
 #include <format>
-#include <thread>
-#include <openssl/evp.h>
 #include <fstream>
+#include <map>
+#include <openssl/evp.h>

```

Comment:
Separate includes into apparent groups.
Prefer order 
- in-house
- standard
- 3rd-party

and sort alphabetically.